public/scripts/app.js

```diff
@@ -17,109 +13,112 @@ $(document).ready(function(){
 
 // Phrase 0: (Adding initial static elements via ajax)
 	
-
-	// $.ajax({
-	// 	method: "GET",
-	// 	url: "/api/profile"
-		
-	// })
-	// .then(function(data){
-	// 	console.log("hello");
-	// 	console.log(data);
-	// 	renderIndex(data);
-
-	// })
-	// .catch(function(err){
-	// 	console.log(err);
-	// });
```

Comment:
Your comments are great! It makes it really easy to understand what you are doing. Keep it up!